test/context/onerror.js

```diff
@@ -248,7 +281,7 @@ describe('ctx.onerror(err)', () => {
       });
 
       app.use(async ctx => {
-        throw {key: 'value'}; // eslint-disable-line no-throw-literal
+        throw { key: 'value' }; // eslint-disable-line no-throw-literal
```

Comment:
Looks like my editor fixed some lint warnings automatically, please tell me if you'd prefer that I revert that.